src/services/car/defaultListing.ts

```diff
@@ -0,0 +1,97 @@
+import { WithValidationError } from "../../types/withValidationError"
+import {
+  DealerDefaultListing,
+  DealerDefaultAdditionalServices,
+  DealerDefaultWarranty,
+} from "../../types/models/listing"
+import {
+  ApiCallOptions,
+  fetchPath,
+  handleValidationError,
+  putData,
+} from "../../base"
+
+export const fetchDealerDefaultListing = async ({
```

Comment:
I know we name our function after backend paths, but I'd expect it to return `Listing` when I read the name.
How about:
```suggestion
export const fetchDealerDefaultListingData = async ({
```